capitals.rb

```diff
@@ -144,10 +145,101 @@
 }, {
     name: "West Virginia",
     capital: "Charleston"
-}, {
-    name: "Wisconsin",
-    capital: "Madison"
-}, {
+},
+   {
+     name: "Wisconsin",
+      capital: "Madison"
+   },
+{
     name: "Wyoming",
     capital: "Cheyenne"
 }]
+
+# gets the first argument
+user_name = ARGV[0]
+prompt = '> '
+
+puts "
+    ###############################################
+
+    Do you know your states?
+
+    ##############################################
+
+    "
+
+puts "Welcome, just enter the name of the Capital for each state with the first letter capitalized, Please enter your name:"
+
+
+#get the users name
+puts prompt
+ user_name = $stdin.gets.chomp
+
+puts "Hi #{user_name}."
+
+
+
+# shuffle the states and start all the scores at zero
+  states.shuffle!
+  $correct = 0
```

Comment:
Consider building this app so that each state-capital pair has its own correct and wrong values. That way, you can avoid defining global variables like this.